index.html

```diff
@@ -12,6 +12,15 @@
   <meta property="og:site_name" content="Email Markup Consortium" />
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://emailmarkup.org"/>
+  <style>
+    body {
+      max-width: 80ch;
+      margin: auto;
+      font-size: 16px;
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
Not a blocker but does `sans-serif` break the stack and stop from rendering e.g. Apple Color Emoji because everyone will have `sans-serif'?

Reply:
nice catch, moving it to the end of the stack

Reply:
I'd already merged this so nbd for now

Reply:
Actually I think this is the proper way to declare font stacks with emojis fonts (see https://css-tricks.com/snippets/css/system-font-stack/ for example). (Not sure why exactly… my understanding is this will allow the `sans-serif` font to be picked up if it has the emojis in it, and only apply the specific emoji font otherwise.)

Reply:
TIL. Thanks Remi.